packages/forms-web-app/src/dynamic-forms/s78-lpa-statement/journey.js

```diff
@@ -38,7 +38,11 @@ class S78LpaStatementJourney extends Journey {
 		// const questionsHaveAnswers = questionsHaveAnswersBuilder(response);
 
 		this.sections.push(
-			new Section('', config.dynamicForms.DEFAULT_SECTION).addQuestion(questions.lpaStatement)
+			new Section('', config.dynamicForms.DEFAULT_SECTION)
+				.addQuestion(questions.lpaStatement)
+
+				//make conditional on additional docs question when added to journey
+				.addQuestion(questions.uploadLpaStatementDocuments)
```

Comment:
if you rebase this branch you'll see I've moved this stuff out of the constructor. Should be easy to port this change, just move it up into the buildSections function